### PR TITLE
Mention stdlib docs, or lack thereof

### PR DESCRIPTION
**Pull request by Tetralux**
Also points out that the documentation in the downloads section also functions as the quick reference guide.

I can remove that if preferred though.

### CODE REVIEW

**Comment by hryx** [comment]
This question comes up multiple times a week, so thanks for addressing it. But there are problems with this:

- The README is meant as a quickstart guide. (This was mentioned in another issue, but I can't find it.) This belongs in [the wiki](https://github.com/ziglang/zig/wiki). I've been meaning to add this as an FAQ there, and looks like it's high time.
- stdlib documentation is planned for before 1.0. The ticket is #21, which has milestone set to 0.5. The 1.0 release is far off still, so this may unnecessarily discourage people.

**Comment by hryx** [comment]
open for bidness: https://github.com/ziglang/zig/wiki/FAQ

**Comment by Tetralux** [comment]
@hryx 
It wouldn't immediately occur to me to search the wiki, so I'd suggest having a reminder in the readme that mentions it. Maybe that's just me.
But yes - other than that sounds good to me.

**Comment by hryx** [comment]
> It wouldn't immediately occur to me to search the wiki

That's totally fair. Presumably the new FAQ will be search engine-friendly, but who knows. Maybe we should link to the FAQ in the "Resources" section? 🤷‍♀

File: README.md [hunk 1]
```diff
@@ -6,10 +6,20 @@ Zig is an open-source programming language designed for **robustness**,
...
+ * [Download & Documentation](https://ziglang.org/download) (These docs function as the quick-reference guide.)
  * [Community](https://github.com/ziglang/zig/wiki/Community)
  * [Contributing](https://github.com/ziglang/zig/blob/master/CONTRIBUTING.md)
 
+## Standard Library Docs
+
+Documentation for the standard library doesn't exist yet, though there will be full docs by version 1.0.
+For now, you can search through `std/` for items of interest with a text editor.
+
+It isn't a priority right now as the focus on more on implementing features and getting stuff working well.
```
Comment (komuw) [inline README.md]:
```suggestion
It isn't a priority right now as the focus is more on implementing features and getting stuff working well.
```

**Comment by andrewrk** [comment]
Thanks for the PR. I took care of this in 04b32d6a8aa30cc894f3057581ffda28466e8ef2.

**Pull request closed without merging**